### PR TITLE
Avoid going to Dashboard on RPi models below 2B

### PR DESCRIPTION
**Pull request by skarbat**
* In such cases, go directly to Desktop mode
 * Having the check on uixinit will work during boot and Greeter login

### CODE REVIEW

**Comment by skarbat** [comment]
This should complete the control to stop RPi models below 2B from going into dashboard mode.
Should work from boot time, Dashboard desktop icon, and the greeter.

@tombettany @alex5imon

File: bin/kano-dashboard-confirm [hunk 1]
```diff
@@ -7,14 +7,28 @@
...
 
-kano-dialog title="Dashboard Mode" \
+# If the kit is running on a RaspberryPI model below 2B,
+# do not switch, because the Dashboard is not supported.
+is_rpi2_or_above=`python -c "from kano.utils.hardware import has_min_performance,\
+  RPI_2_B_SCORE ; print has_min_performance(RPI_2_B_SCORE)"`
+
+if [ "$is_rpi2_or_above" != "True" ]; then
+    kano-dialog title="Dashboard no supported" \
+        description="Please, consider upgrading to a RaspberryPI 2 or above." \
```
Comment (Ealdwulf) [inline bin/kano-dashboard-confirm]: Might want to ask the designers what they want for this piece of text.

**Comment by Ealdwulf** [comment]
Looks good @skarbat :+1:

**Comment by skarbat** [comment]
Thanks @Ealdwulf , updated text and merging now.

**Pull request merged**